### DISCUSSION TITLE
Uso de qsort en la T3

### DISCUSSION THREAD

**Discussion by JavierLarre** [IIC2133-PUC/2024-1#181] [Tarea 3] [answered]
Hola, en la librería `stdlib.h` de C hay una función qsort para ordenar un arreglo con un criterio dado. Si en la tarea necesito ordenamiento entonces puedo ocupar qsort o tengo que implementar mi función propia? gracias.

**Comment by 4n1t4** [comment, marked as answer]
Hola!
Dado que el propósito de la tarea no abarca evaluar su capacidad de codificar algoritmos de ordenación es que SI está permitido utilizar librerias para ello.
